tutorials/notebooks/example_pytorch_mobilenetv2_cifar100_mixed_precision.ipynb

```diff
@@ -0,0 +1,663 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "7cf96fb4",
+   "metadata": {
+    "id": "7cf96fb4"
+   },
+   "source": [
+    "# Mixed-Precision PTQ - Pytorch MobileNetV2 on CIFAR100"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "59ed8f02",
+   "metadata": {
+    "id": "59ed8f02"
+   },
+   "source": [
+    "[Run this tutorial in Google Colab](https://colab.research.google.com/github/reuvenperetz/model_optimization/blob/add-cifar100-notebook/tutorials/notebooks/example_pytorch_mobilenetv2_cifar100_mixed_precision.ipynb)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "822944a1",
+   "metadata": {
+    "id": "822944a1"
+   },
+   "source": [
+    "## Overview"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "743dbc3d",
+   "metadata": {
+    "id": "743dbc3d"
+   },
+   "source": [
+    "This tutorial demonstrates the process of retraining and quantizing a MobileNetV2 on CIFAR100 dataset. It starts by fine-tuning a pretrained MobileNetV2 model on the CIFAR100 dataset. After retraining, the model is quantized using MCT. This tutorial specifically uses mixed-precision quantization, which assigns different precision levels to different layers in the model based on their impact on the output. The quantized model is then evaluated and exported to an ONNX file."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "59e2eeae",
+   "metadata": {
+    "id": "59e2eeae"
+   },
+   "source": [
+    "## Summary"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1daf577a",
+   "metadata": {
+    "id": "1daf577a"
+   },
+   "source": [
+    "In this tutorial we will cover:\n",
+    "1. Retraining Pytorch MobileNetV2 on CIFAR100.\n",
+    "2. Quantizing the model using post-training quantization in mixes-precision for the weights.\n",
+    "3. Evaluating and exporting the model to ONNX."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8b3396bf",
+   "metadata": {
+    "id": "8b3396bf"
+   },
+   "source": [
+    "## Setup"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5e7690ef",
+   "metadata": {
+    "id": "5e7690ef"
+   },
+   "source": [
+    "First install the relevant packages and import them:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "89e0bb04",
+   "metadata": {
+    "id": "89e0bb04"
+   },
+   "outputs": [],
+   "source": [
+    "! pip install -q model-compression-toolkit\n",
+    "! pip install -q mct-quantizers==1.1.0\n",
```

Comment:
Isn't this part of the model-compression-toolkit requirements?
I just prefer to limit the amount of places where we specify the exact mct-quantizers version, so we won't need to update it every time we release a new version

Reply:
It will be removed after 1.9.1 release